src/Api/API.php

```diff
@@ -3325,6 +3325,15 @@ protected function applyMassiveAction(
         $action = explode(':', $action_key);
         $processor = $action[0];
 
+        $params['field'] = array_keys($params)[0];
+
+        foreach (Search::getOptions($itemtype) as $key => $value) {
+            if ($value['field'] == $params['field']) {
+                $params['search_options'][$itemtype] = $key;
+                break;
+            }
+        }
```

Comment:
Maybe this piece of code should be nested into a `if ($action[1] === 'update') { }` block.